apis/sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/write/WriteBridgeService.java

```diff
@@ -592,8 +593,18 @@ private Uni<ResultSet> executeBatch(
                             .setValue(queriesConfig.consistency().writes())));
     batchQueries.forEach(batch::addQueries);
     Batch batchBuilt = batch.build();
-    if (logger.isDebugEnabled()) {
-      logger.debug("Batch payload size : {}", batchBuilt.getSerializedSize());
+    if (batchBuilt.getSerializedSize() >= BATCH_PAYLOAD_SIZE_LIMIT) {
+      logger.warn(
+          "Tenant: {}, Batch payload size : {} in bytes, Number of CQL Statements : {}",
```

Comment:
Uhh. And I forgot to add what I really meant to ask: something to indicate this is "too big" batch -- so something like "Batch bigger than expected maximum size {}: ...."

Just so message is clear on why the warning is being logged (as opposed to regular `debug` below).

Reply:
Done.